### ISSUE TITLE
[ui5-page | angular]: Uncaught DOMException: Failed to construct 'CustomElement': The result must not have attributes

### ISSUE THREAD

**Issue by rabbal** [SAP/ui5-webcomponents#9981]
### Bug Description

After upgrading to v2.3.0 with angular 18.2.7, we faced an error that says the ui5-page has attributes before attaching to DOM. After further investigation, I realized that the problem is related to changes in the https://github.com/SAP/ui5-webcomponents/pull/9885 merge request in which you [added the style attribute to ui5-page in its constructor](https://github.com/SAP/ui5-webcomponents/commit/e8beb76ea28f76eb39af40df9c0cc5c603a365d2#diff-9efdd92722f18513b127bfb656da3734f7025c49e4a5cb5cd15d10944a813a84R119).

### Affected Component

ui5-page

### Expected Behaviour

Render as the previous version in angular

### Steps to Reproduce

1. npm install -g @angular/cli
2. ng new ui5-web-components-application
3. cd ui5-web-components-application
4. npm install @ui5/webcomponents-fiori
5. clear the content of app.component.html
6. import '@ui5/webcomponents-fiori/dist/Page.js' inside main.ts
7. utilize ui5-page inside app.component.html
8. ng s -o
9. you'll face the mentioned error and Angular won't create the respective element and render it

### Priority

Very High

### UI5 Web Components Version

2.3.0

### Browser

Chrome

### Operating System

Ubuntu

### Additional Context
If you comment on the mentioned line in the constructor, the issue will disappear. Also, it is recommended that such things be moved into connectedCallback.

_No response_

### Declaration

- [X] I’m not disclosing any internal or sensitive information.

**Comment by hinzzx** [comment]
Hello, 

@SAP/ui5-webcomponents-topic-rl ,

It seems that after setting an attribute to the `ui5-page` web component in the constructor, an Error is thrown in an Angular application.

![image](https://github.com/user-attachments/assets/f27bdb3f-f909-4df9-a289-ae108978577c)

The issue is easily reproducible by the given steps.

Note: 
Don't forget to add 
**CUSTOM_ELEMENTS_SCHEMA**, in the app.component.ts file

```diff
+ import { Component, CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
import { RouterOutlet } from '@angular/router';
import '@ui5/webcomponents-fiori/dist/Page.js';

@Component({
  selector: 'app-root',
  standalone: true,
  imports: [RouterOutlet],
  templateUrl: './app.component.html',
  styleUrl: './app.component.css',
+schemas: [CUSTOM_ELEMENTS_SCHEMA]  // Allow custom elements in the application
})
export class AppComponent {
  title = 'ui5-web-components-application';
}
```

Best Regards,
Stoyan

**Comment by stephen-schmith** [comment]
@nikoletavnv Has this fix been released yet? I'm still seeing this error in 2.3.0.

**Comment by nikoletavnv** [comment]
Hello @stephen-schmith, 
The fix will be included in the next sub version - 2.4.0 of the web components.

Kind regards, 
Nikoleta
